CMakeLists.txt

```diff
@@ -97,6 +97,9 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
   set (WARN_FLAGS "${WARN_FLAGS} -Wno-missing-noreturn -Wno-unknown-pragmas")
   set (WARN_FLAGS "${WARN_FLAGS} -Wno-gnu-zero-variadic-macro-arguments")
   set (WARN_FLAGS "${WARN_FLAGS} -Wconversion")
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "13.0")
```

Comment:
Got it. So, `VERSION_GREATER` -> `VERSION_GREATER_EQUAL` is the change we needed?

Reply:
> Got it. So, `VERSION_GREATER` -> `VERSION_GREATER_EQUAL` is the change we needed?

True, sorry for the careless mistake.